ultimate-pipeline/src/ultimate_pipeline/pipelines/data_processing/supervisely_converter.py

```diff
@@ -191,16 +191,21 @@ def extract_boxes_from_graph(detected:dict) -> Generator[Tuple[int, np.ndarray,
             if node_key in detected_nodes:
                 if "disabled" in detected_nodes[node_key]:
                     continue
+
+                #class_idx = graph_id_to_idx[class_id]
+                # HACK - hardcoded index start
+                class_idx = 31 + i
+                # Cosmological constant :)
+                fudge_factor = 1.5 if class_idx >= 38 else 1.0
```

Comment:
A quick hack to make bounding boxes at the front bigger than those at the back :)
(would be good to make it a proper parameter)